query_optimizer/tests/execution_generator/Select.test

```diff
@@ -632,6 +632,48 @@ ORDER BY group_col1,
 +--------------------+--------------------+--------------------+
 ==
 
+
+CREATE TABLE dates (value DATETIME);
```

Comment:
If we change one test file in `query_optimizer`, we need to update other three test files under different directories. In this case, they are

 * `resolver`
 * `logical_generator`
 * `physical_generator`